src/fable-loader/index.js

```diff
@@ -35,10 +35,12 @@ function getCompiler(webpack, args) {
 
 function transformBabelAst(babelAst, babelOptions, sourceMapOptions, callback) {
     var fsCode = null;
+    var fileName = sourceMapOptions.path.replace(/\\/g, '/');
```

Comment:
This line should probably be after the ` if (sourceMapOptions != null)` check

Reply:
Thanks for pointing that out @xdaDaveShaw! I'll make the change :+1:

Reply:
Oh man, can't believe I missed that! The tunnel vision is real 😮